juniper_official/Solutions/Traffic-Blackhole/pipe-stats.rule

```diff
@@ -70,7 +71,7 @@ healthbot {
                     path "/components/component/@name";
                 }
                 type string;
-		description "FPC name";
+                description "FPC name";
```

Comment:
For keeping common names, can we rename this to "fpc"? Keys section also should be adjusted accordingly

Reply:
Changed to fpc